ml-dhs-aws/src/test/java/com/marklogic/client/dhs/test/TestJobRunner.java

```diff
@@ -30,174 +28,204 @@
 import static org.junit.Assert.fail;
 
 public class TestJobRunner {
-   static final private Logger logger = LoggerFactory.getLogger(TestJobRunner.class);
-   static final private String csvFile = "data"+File.separator+"OrderLines.csv";
-   static final private String role = "rest-tracer";
+	static final private String csvFile = "data" + File.separator + "OrderLines.csv";
+	static final private String role = "rest-tracer";
+
+	DatabaseClient client;
+
+	public class TestJobLogger implements JobLogger {
+		int countSuccesses = 0;
+		int countFailures = 0;
+
+		@Override
+		public void status(ObjectNode obj) {
+			countSuccesses++;
+			assertNotNull(obj);
+			assertNotNull(obj.path("Timestamp"));
+			assertNotNull(obj.path("Current status"));
```

Comment:
Where the "Current status" property indicates that we're logging batch success, could the test verify that the expected properties for a batch status message are not null?  The motivation would be to confirm that the status logging is providing what a progress indicator would need.